app/src/main/java/com/github/houseorganizer/houseorganizer/util/Util.java

```diff
@@ -56,5 +60,14 @@ public static void displayErrorMessage(ErrorType err, TextView error_field) {
         }
     }
 
+    //<---------------------| Connection Status |------------------------------------------->
+    public static boolean hasWifiOrData(Context panelCtx) {
+        ConnectivityManager connectivityManager =
+                (ConnectivityManager) panelCtx.getSystemService(Context.CONNECTIVITY_SERVICE);
```

Comment:
So that's how we do it

Reply:
Yeah - to be fair we should ask for a permission related to this. Should be available next sprint